integration_tests/payloads/filterFunctionpayload.json

```diff
@@ -0,0 +1,21 @@
+{
+  "Records": [
+    {
+      "messageId": "a81a7828-579c-4958-b67a-63f81b997f61",
+      "receiptHandle": "AQEBR9preCLGnSoKG64NF+pQ6xkVsST+g3cHYz1MQvP0Xu+0BWv466jrMIVkoMf1eBICI0nBz2eTZaqoxoearK7/3+Nkktgc3Dgo9u7l4r4yRgKHEx6htKHuaOZInpRejZoEjBxWz2sH2B1FONLmo16V6r+8qqCmCeXbfhxKqirpUymb1mTgnTRtQvyAgvexHy+88RQbCxOC2Pcjkg4+qjmaJrW085pyTgtMIdtPAKeFVmxek5PvkgzpYj+vcemBjhJJQQTd1magk2XS4uQvhxe0+vRx6pmFIsCnGjaZHZgWLVDcY6ibA4NA0qxLcgBPt5QAVC5DIg35feTOraAWFD1cHxC3VDfhnAIoSs+HvI6LPljfKK+5C1/c8t2EZYVAyOAIHznQkFpyym0ktEa3/jNHiQ==",
```

Comment:
I haven't checked in depth, but does this event need to be so realistic here? We're probably not touching most of these fields in our functions. Maybe put some text in there that indicates that? Like 
`"receiptHandle": "some internal handle"`

Reply:
yes no need to be realistic added some text for the unused fields